src/main/java/FileHandler.java

```diff
@@ -0,0 +1,79 @@
+package main.java;
+
+import main.java.task.*;
```

Comment:
Avoid using wildcard imports. You can tweak the settings in IntelliJ IDEA to avoid wildcard imports because they have some default number of before they start converting them to wildcard imports.

You can also try searching for ".*" using your IDE to check for all wildcard imports to make sure they are all removed.